sample-extension/extension-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "commerce-first-app",
+  "displayName": "Commerce First App on App Builder",
```

Comment:
Maybe it's better to make it more specific, like `Adobe Commerce` instead of just `Commerce`